src/patternfly/components/Switch/docs/code.md

```diff
@@ -0,0 +1,28 @@
+## Overview
+
+A Switch is an alternative to the checkbox component.
+
+Use a switch when your user needs to perform instantaneous actions without confirmation.
+
+Use checkbox if your user has to perform additional steps for changes to become effective.
+
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `role="switch"` | `.pf-c-switch` |  Identifies the element as a switch. **required** |
+| `aria-label="..."` | `.pf-c-switch` |  Indicates the action triggered by the switch. **required**  |
+| `aria-checked="true/false"` | `.pf-c-switch` |  Indicates that the switch is on/off. **required**  |
+| `aria-disabled="true"` | `.pf-c-switch__input` |  Indicates that the input is disabled |
```

Comment:
Related to my other comment, we can remove this row. We only need the `disabled` attribute.

Reply:
aria-checked="true" is mandatory by the travis accessibility tests

Reply:
What I was suggesting was to remove `aria-disabled`—sorry for not being clear in my previous comment. 

But now that you mention it, that does make sense that `aria-checked` would be required for `role="switch"`. If the goal of using `<input type="checkbox">` is to get behavior and interaction for free without additional JS, then we might want to drop `role="switch"`. Otherwise, we would require JS to handle setting the correct state in `aria-checked`. This probably isn't a big deal in the React implementation, but there's a risk that anyone not using the React implementation would overlook this. And I don't think there's a huge benefit in using `role="switch"` versus the implicit role of the checkbox.

Reply:
done